### PR TITLE
🐛⚰️♻️📦🚨Feature/compose update

### PR DESCRIPTION
**Pull request by abunur**
Updates and fixes for Compose Sample
- gets app to compile
- gets app to actually run
- fixes all the issues
- now matches the other todo apps (non-compose on Android, iOS app)

- 📦Gradle, library updates
- ⚰️remove dead code
- ♻️code clean up
- 🐛fix upsert
- 🐛fix delete

### CODE REVIEW

File: tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/TasksApplication.kt [hunk 1]
```diff
@@ -3,20 +3,33 @@ package live.ditto.compose.tasks
...
     override fun onCreate() {
         super.onCreate()
+        setupDitto()
+    }
+
+    private fun setupDitto() {
         val androidDependencies = DefaultAndroidDittoDependencies(applicationContext)
-        val identity = DittoIdentity.OfflinePlayground(appID = "YOUR_APP_ID", dependencies = androidDependencies);
-        ditto = Ditto(androidDependencies, identity)
+        // Create an instance of DittoSyncKit
```
Comment (phatblat) [inline tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/TasksApplication.kt:21]: The `DittoSyncKit` name is obsolete; it's just "Ditto" now.

File: tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/MainActivity.kt [hunk 2]
```diff
@@ -20,10 +21,9 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        val ditto = TasksApplication.ditto
         try {
-            ditto!!.setOfflineOnlyLicenseToken("<REPLACE_ME>")
-            ditto!!.startSync()
+//            ditto?.setOfflineOnlyLicenseToken("<REPLACE_ME>")
```
Comment (phatblat) [inline tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/MainActivity.kt:25]: If this app is only going to use `OnlinePlayground` then this is dead code.

**Review by phatblat — APPROVED** [review]

File: tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/TasksApplication.kt (hunk 1 above)
Comment (abunur) [reply tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/TasksApplication.kt]: removed

File: tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/MainActivity.kt (hunk 2 above)
Comment (abunur) [reply tasks/jetpack-compose/app/src/main/java/live/ditto/compose/tasks/MainActivity.kt]: ⚰️removed

**Pull request merged**